CRM/Logging/Differ.php

```diff
@@ -142,14 +141,21 @@ public function diffsInTable($table, $contactID = NULL) {
       }
     }
 
+    $logDateClause = '';
+    if ($this->log_date) {
+      $params[2] = array($this->log_date, 'String');
+      $logDateClause = "
+        AND lt.log_date BETWEEN DATE_SUB(%2, INTERVAL {$this->interval}) AND DATE_ADD(%2, INTERVAL {$this->interval})
```

Comment:
Ok - I've added something to sanitise it - the normal DAO escaping can't work as it would wrap it in undesirable appostrophes